pandapipes/multinet/control/run_control_multinet.py

```diff
@@ -260,6 +267,16 @@ def prepare_run_ctrl(multinet, ctrl_variables):
         ctrl_variables[net_name]['only_v_results'], ctrl_variables[net_name]['recycle'] = \
             get_recycle(ctrl_variables_net)
 
+        if ('continue_on_divergence') in kwargs and (ctrl_var is None):
```

Comment:
Same as in the comment above. I'd expect something like `and ((ctrl_var is None) or (continue_on_divergence not in ctrl_var.keys()))`